src/server/routes/createInternalProxies.integration.spec.ts

```
@@ -0,0 +1,116 @@
+import { createFakeUpstream } from '../../__test-utils__/createFakeUpstream';
+import { createTestApp, sendTestRequest } from '../../__test-utils__/createTestApp';
```

ESLint

[error] Replace `·createTestApp,·sendTestRequest·` with `⏎··createTestApp,⏎··sendTestRequest,⏎`

(prettier/prettier)